libs/libmaxminddb/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libmaxminddb
-PKG_VERSION:=1.7.1
+PKG_VERSION:=1.9.1
 PKG_RELEASE=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/maxmind/libmaxminddb/releases/download/$(PKG_VERSION)/
```

Comment:
Removing / is intentional?

Reply:
I think it was instinctive. But I've just checked - out of 1139 PKG_SOURCE_URL in this repo only 235 has `/` in the end.